src/appleseed-max-impl/appleseedrenderer/appleseedrendererparamdlg.cpp

```diff
@@ -712,8 +716,14 @@ namespace
             m_spinner_renderingthreads->SetResetValue(RendererSettings::defaults().m_rendering_threads);
             m_spinner_renderingthreads->SetValue(m_settings.m_rendering_threads, FALSE);
 
+            const wchar_t* log_combo_items[] = { L"Always", L"Never", L"On Error" };
```

Comment:
Mark as `static`. Also, it's an array of constants so `LogComboItems`, according to our conventions.